receiver/mysqlreceiver/documentation.md

```diff
@@ -26,6 +26,8 @@ These are the metrics available for this scraper.
 | **mysql.row_locks** | The number of InnoDB row locks. | 1 | Sum(Int) | <ul> <li>row_locks</li> </ul> |
 | **mysql.row_operations** | The number of InnoDB row operations. | 1 | Sum(Int) | <ul> <li>row_operations</li> </ul> |
 | **mysql.sorts** | The number of MySQL sorts. | 1 | Sum(Int) | <ul> <li>sorts</li> </ul> |
+| **mysql.statement_event.count** | Summary of current and recent statement events. | 1 | Sum(Int) | <ul> <li>schema</li> <li>digest</li> <li>digest_text</li> <li>event_states</li> </ul> |
+| **mysql.statement_event.wait.time** | The total wait time of the summarized timed events. | ns | Sum(Int) | <ul> <li>schema</li> <li>digest</li> <li>digest_text</li> </ul> |
```

Comment:
Shouldn't be `mysql.statement_event.wait_time` or `mysql.statement_event.time`?

Reply:
cc @djaglowski

Reply:
I think the current name is appropriate. 

Including `wait` is an important distinction about what is being measured. Without it, I think it'd be too easily mistaken for a measurement of statement execution time.

Since `.time` metrics are called out specifically in the spec's naming guidelines, I think `wait.time` is appropriate. There is another reason though, which is that "waits" (in some contexts, but maybe not this one) are events that can also be counted, which would then be able to share a namespace. eg. `mysql.statement_event.wait.count`.

Reply:
@dmitryax, any objections to moving forward with the current name based on the above?

Reply:
no objections